articles/azure-arc/servers/manage-vm-extensions.md

```diff
@@ -54,6 +54,7 @@ Arc-enabled servers support moving machines with one or more VM extensions insta
 |Extension |Publisher |Type |Additional information |
 |----------|----------|-----|-----------------------|
 |Azure Defender integrated vulnerability scanner |Qualys |WindowsAgent.AzureSecurityCenter |[Azure Defender’s integrated vulnerability assessment solution for Azure and hybrid machines](../../security-center/deploy-vulnerability-assessment-vm.md)|
+|Microsoft Antimalware extension |Microsoft.Azure.Security |IaaSAntimalware |[Microsoft Antimalware Extension for Windows](../../virtual-machines/extensions/iaas-antimalware-windows)
```

Comment:
Missed changing the product name in the Additional information column - [Microsoft Antimalware Extension for Windows] to [Microsoft Antimalware extension for Windows].  Thanks!

Reply:
Also you didn't fix the path to the article by adding the .md extension to the article name (that is, iaas-antimalware-windows.md)

Reply:
My mistake. Fixing that now!